### PR TITLE
#1 added liquibase changesets for creation user-related tables

### PR DESCRIPTION
**Pull request by MaryR-code**
1. Сreated roles table
2. Сreated user_roles table
3. Added foreign keys to user_roles table

### CODE REVIEW

File: src/main/resources/db/changeLog-0001-user.xml [hunk 1]
```diff
@@ -28,5 +28,34 @@
...
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    
+    <changeSet id="add foreign keys to user_roles table" author="Maria Romanovskaia">
+        <addForeignKeyConstraint baseTableName="user_roles" baseColumnNames="user_id" constraintName="user_id"
+                                 referencedTableName="users" referencedColumnNames="id"/>
+        <addForeignKeyConstraint baseTableName="user_roles" baseColumnNames="role_id" constraintName="role_id"
+                                 referencedTableName="roles" referencedColumnNames="id"/>
```
Comment (dima-fedotovs) [inline src/main/resources/db/changeLog-0001-user.xml:58]: It is Ok to keep it as separate change set. But also it is possible to combine along with table creation. I mean something like:

    <createTable ....>
        <column ....>
            <constraints nullable="false" 
                         foreignKeyName="fk_user_roles_user_id" 
                         referencedTableName="users" referencedColumnNames="id"/>
         </column>
     </createTable>

File: src/main/resources/db/changeLog-0001-user.xml [hunk 2]
```diff
@@ -28,5 +28,34 @@
...
+    </changeSet>
+
+    <changeSet id="create user_roles table" author="Maria Romanovskaia">
+        <createTable tableName="user_roles">
+            <column name="user_id" type="uuid">
+                <constraints nullable="false"/>
+            </column>
+            <column name="role_id" type="uuid">
+                <constraints nullable="false"/>
+            </column>
```
Comment (dima-fedotovs) [inline src/main/resources/db/changeLog-0001-user.xml:50]: In fact, tables without a primary key are not welcome :)
In this particular case the uniqueness of the table consists of both columns. So, both of them can be marked as primary keys

**Review by dima-fedotovs — CHANGES_REQUESTED** [review]
Please add primary keys constraints to user_roles table

File: src/main/resources/db/changeLog-0001-user.xml (hunk 2 above)
Comment (MaryR-code) [reply src/main/resources/db/changeLog-0001-user.xml]: done

**Review by dima-fedotovs — APPROVED** [review]
Everything is all right. May be merged.

**Pull request merged**